src/libs/dates.ts

```diff
@@ -60,6 +60,15 @@ export const formatToSlashedFrenchDate = (ISODate: string) => {
   return `${day}/${month}/${year}`
 }
 
+export const setOneDayEarlierAndFormatToSlashedFrenchDate = (ISODate: string) => {
+  const date = new Date(ISODate)
+  date.setDate(date.getDate() - 1)
+  const day = ('0' + date.getDate()).slice(-2)
+  const month = ('0' + (date.getMonth() + 1)).slice(-2)
+  const year = date.getFullYear()
+  return `${day}/${month}/${year}`
+}
```

Comment:
Je sais que c'est temporaire, mais pourquoi ne pas reprendre l'existant ?

```suggestion
export const setOneDayEarlierAndFormatToSlashedFrenchDate = (ISODate: string) => {
  const date = new Date(ISODate)
  date.setDate(date.getDate() - 1)
  return formatToSlashedFrenchDate(date)
}
```